From 92595bccaf9568bead4fd9b32fa6e09d9357f5b7 Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Fri, 13 Feb 2015 04:58:40 +0800
Subject: [PATCH 0242/1594] MLK-10266-1 mxc: capture fix build error

commit b23d63b853c2249c04fc5fa7589c7152944d7793 from
git://git.freescale.com/imx/linux-2.6-imx.git

cherry-pick some driver tree patch.
change interface funcation name.

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 drivers/media/platform/mxc/subdev/mx6s_capture.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/platform/mxc/subdev/mx6s_capture.c b/drivers/media/platform/mxc/subdev/mx6s_capture.c
index df4efcf..24747c7 100644
--- a/drivers/media/platform/mxc/subdev/mx6s_capture.c
+++ b/drivers/media/platform/mxc/subdev/mx6s_capture.c
@@ -1607,7 +1607,7 @@ static int mx6sx_register_subdevs(struct mx6s_csi_dev *csi_dev)
 		port = of_get_next_child(node, NULL);
 		if (!port)
 			continue;
-		rem = v4l2_of_get_remote_port_parent(port);
+		rem = of_graph_get_remote_port_parent(port);
 		of_node_put(port);
 		if (rem == NULL) {
 			v4l2_info(&csi_dev->v4l2_dev,
-- 
1.7.5.4

